src/main/java/org/folio/circulation/resources/RequestCollectionResource.java

```diff
@@ -228,19 +238,53 @@ void getMany(RoutingContext routingContext) {
 
     final var itemRepository = new ItemRepository(clients);
     final var userRepository = new UserRepository(clients);
+    final var servicePointRepository = new ServicePointRepository(clients);
     final var loanRepository = new LoanRepository(clients, itemRepository, userRepository);
     final var requestRepository = RequestRepository.using(clients,
       itemRepository, userRepository, loanRepository);
-
-    fromFutureResult(requestRepository.findBy(routingContext.request().query()))
-      .map(this::mapToJson)
+    final var encodedQuery = routingContext.request().getParam("effectiveLocationPrimaryServicePointId");
```

Comment:
"effectiveLocationPrimaryServicePointId" should probably be a constant, as it is referred to in multiple places.